src/lib/utils.js

```diff
@@ -0,0 +1,49 @@
+import { MissionUtils } from '@woowacourse/mission-utils';
+import {
+  ERROR_MESSAGE_NOT_INTEGER,
+  ERROR_MESSAGE_NOT_NUMBER,
+  ERROR_MESSAGE_NOT_POSITIVE_POSITIVE,
+} from './constants.js';
+
+export const readLineAsync = async message =>
```

Comment:
여기에 `async`를 작성하신 이유가 있으실까요? 별도의 로직이 없으니 `async`가 없어도 똑같이 동작할 것 같습니다..!
```suggestion
export const readLineAsync = message =>
```

Reply:
그렇군요 ,,!! 

async로 감싸야 프로미스를 반환할 줄 알았는데, `MissionUtils.Console.readLineAsync`가 프로미스를 반환하기 때문에 async로 감싸지 않아도 되는군요. async에 대해 더 공부해봐야겠네요. 감사합니다

Reply:
저두 최근에 알았답니다 ㅎㅎ 저도 비동기면 무작정 `async`로 감싸기 일쑤였어요 😂